db/migrate/20150904181202_miq_groups_add_ldap_role.rb

```diff
@@ -0,0 +1,17 @@
+class MiqGroupsAddLdapRole < ActiveRecord::Migration
+  class MiqUserRole < ActiveRecord::Base
+  end
+
+  class MiqGroup < ActiveRecord::Base
+  end
```

Comment:
What part @matthewd?  I can't tell if you're saying that this shouldn't be necessary or something else.